misp_stix_converter/stix-to-misp.py

```diff
@@ -1,69 +1,94 @@
 #!/usr/bin/env python3
-
-##########################
-#     WRAPPER SCRIPT     #
-#     DEALS WITH STIX    #
-#     TO MISP            #
-##########################
-
+"""Wrapper script providing STIX to MISP conversion. """
 
 import argparse
 import logging
-import pyaml
-import time
-import sys
 import os
+import sys
+import time
+
+import pyaml
 
-from misp_stix_converter.servers import misp
 from misp_stix_converter.converters.convert import load_stix
+from misp_stix_converter.servers import misp
+
+LOGGER_FORMAT = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
```

Comment:
Kinda for lines 15-20

If they're constants, why bother defining them beforehand when they can be passed straight to a function without adding more lines?

It executes the same but still, seems unneeded

Reply:
Is of course individual perception, but in my experience, complexity is not in the number of lines. A logger format is some public interface, and might ease adaption and use, if made visible on module level.